api_service/main.py

```diff
@@ -279,17 +266,17 @@ def create_features_and_values(attributes: Dict[str, str]) -> Dict[int, int]:
 
 
 def manage_categories() -> None:
+    # ids = []
+    # for category in prestashop.get("categories")["categories"]["category"]:
+    #     if int(category["attrs"]["id"]) not in [1, 2]:
+    #         ids.append(int(category["attrs"]["id"]))
+    # if ids:
+    #     print("Deleting categories...")
+    #     prestashop.delete("categories", resource_ids=ids)
```

Comment:
nit: move the deletion of categories to a separate function and add a flag e.g.: 
`
def manage_categories(force_remove: bool = False) -> None:
       if force_remove:
             delete_categories()
       ....
`